docs/meshstack.billing-configuration.md

```diff
@@ -3,11 +3,23 @@ id: meshstack.billing-configuration
 title: Configuration
 ---
 
-## Supported Currencies
+## Currency Conversion
 
-In order to simplify multi-cloud governance, cloud foundation teams should prefer standardising on a single chargeback currency.
-However, Cloud Foundation teams sometimes encounter situations in which they need to process billing or chargeback in multiple
-currencies. For example GCP might charge your organization for cloud consumption in EUR while Azure charges consumption in USD.
+To streamline cloud governance, meshStack recommends using a single chargeback currency. This functionality automatically converts $ from cloud providers like AWS into Euro. This simplification aids in the chargeback process to teams. 
+
+You have two options for enabling currency conversion in chargeback and usage reports:
+
+- Standard Currency Rate: By default, meshStack uses the conversion rates from frankfurter.app, which provides exchange rates from the European Central Bank. To activate this, navigate to the Financial Tab on the Settings page and enable the conversion feature using the provided toggle.
```

Comment:
I removed the plans as we are currently in the process of changing the plans and currency conversion will not be part of the differentiation anymore